conan/internal/api/new/bazel_exe.py

```diff
@@ -17,7 +17,7 @@ class {{package_name}}Recipe(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
 
     # Sources are located in the same place as this recipe, copy them to the recipe
-    exports_sources = "main/*", "WORKSPACE"
+    exports_sources = "main/*", "WORKSPACE", ".bazelrc"
```

Comment:
Should'nt this also be conditional? Users will be a bit surprise if this appears here without that file existing